docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
```

Comment:
We already have a docker compose in the packages/commonwealth folder. We should remove this one since our root folder is not related to any specific environment

Reply:
Snuck in there when i was doing merge conflicts, it's taken care of in latest commit